### PR TITLE
Full 2nd edition overhaul

### LINKED ISSUE #47: 2nd Edition
Might as well start on this:

- Should it be a whole separate project or build on this one?
- What icons need to be added?

### LINKED ISSUE #49: Ordnance-type Icon
image

See that little missile icon next to the range limitations of the attack?  That represents an "Ordnance" type of secondary weapon, which ignores range combat bonuses (as opposed to Cannons and Turrets, which now are affected by range).

Just another icon to consider in the grand scheme of things...

### PR DESCRIPTION
**Pull request by Hinny**
- Resize font dimentions to correspond with official 2nd edition rulebook
 - Copy all icons included in official rulebook
 - Remove icons not compatible with 2nd edition
 - Resize and realign all upgrade icons not included in official rulebook
 - Add missing symbols (recurring, linked, range-bonus-indicator, and right and left arc)
 - Update custom maneuvers
 - Update relevant files with new icon refs and delete 1st-edition-only refs
 - Close #47, close #49

### CODE REVIEW

**Comment by sirjorj** [comment]
`Remove icons not compatible with 2nd edition` - what all got removed?  Because I think the intent is to keep all the 1st edition stuff here as well.

And if resize and realign means what I think it means, THANK YOU! :)

**Comment by Hinny** [comment]
No in #47 this was already discussed.
"If the concern is that someone wants to use the old icon set or doesn’t need any new icons, then they should use the last 1.x released version. I’m also assuming we’ll likely start versioning 2.x when 2nd edition is integrated."

**Comment by sirjorj** [comment]
Oh it is splitting.  I missed that.

**Comment by geordanr** [comment]
I wasn't under the impression that there would be a split; the [full comment](https://github.com/geordanr/xwing-miniatures-font/issues/47#issuecomment-391909300) quoted above was

> Duplicating the same icons in two font files seems error prone to me. **I vote to just replace any existing icons that will change (and I don’t know if there actually are any) and add new icons to the same file.** If the concern is that someone wants to use the old icon set or doesn’t need any new icons, then they should use the last 1.x released version. I’m also assuming we’ll likely start versioning 2.x when 2nd edition is integrated.

(emphasis mine)

The implication being we would continue using one TTF, and if someone has a requirement for 1.0 icons they would use an older 1.x version of this project, and not the latest version but use a 1.0 TTF that was distributed within it.

**Comment by Hinny** [comment]
I agree. Not a split, just different version of the same file.

**Comment by Hinny** [comment]
So, should I go on and merge or leave it open a bit longer?

File: src/json/icons-map.json [hunk 1]
```diff
@@ -27,17 +21,19 @@
         "crit": "\u0063",
         "dalan-bankleft": "\u005b",
         "dalan-bankright": "\u005d",
+        "device": "\u0042",
         "doubleturretarc": "\u0071",
-        "elite": "\u0045",
```
Comment (geordanr) [inline src/json/icons-map.json]: Can we keep `elite` as an alias for `talent`?

Comment (Hinny) [reply src/json/icons-map.json]: I need to understand why first? It is called talent in 2nd edition. I renamed bomb to device as well btw.

Comment (geordanr) [reply src/json/icons-map.json]: Actually, on second thought that's fine.

**Pull request merged**